client/branded/src/global-styles/colors-redesign.scss

```
@@ -67,7 +86,7 @@ $theme-colors-redesign: (
 .theme-light.theme-redesign {
     --primary-2: #a3d0ff;
     --primary-4: #e1f0ff;
-    --secondary: #{$redesign-gray-04};
+    --secondary-light: var(--gray-04);
```

```suggestion
    --secondary: var(--gray-04);
```

Otherwise `--secondary` will have no value in `theme-light` I think